src/__tests__/App.test.js

```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import React from 'react';
+import store from '../Redux/store';
+import App from '../App';
+
```

Comment:
- [ ] Kindly add some unit tests for the app's components and some pure functions in the Redux code